exp/controllers/awsmachinepool_controller.go

```diff
@@ -224,6 +252,30 @@ func (r *AWSMachinePoolReconciler) reconcileNormal(_ context.Context, machinePoo
 	// Set state
 	// Reconcile AWSMachinePool State
 	//Handle switch case instance.State{}
+	// switch instance.State {
```

Comment:
these comments needed? i have a feeling they may conflict with #31

Reply:
This was me not pushing 🤦‍♀️